src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -323,38 +340,50 @@ private void validateCvrFileSources() {
     }
   }
 
+  private boolean candidateStringIsAlreadyInUseElsewhere(
+      String candidateString, String field, Set<String> candidateStringsSeen) {
+    boolean foundError = false;
+
+    if (candidateStringsSeen.contains(candidateString)) {
+      isValid = false;
```

Comment:
This should be `foundError = true;` since you already set `isValid` to `false` after calling this function both times.